src/view/cards/paservice_cards/pa_service_content_card.py

```diff
@@ -6,21 +6,32 @@
 
 
 class PaServiceContentCard:
+    type_icon = {
+        "beitrag": "📰",
+        "audio": "🔊",
+        "video": "🎥",
+    }
+
+    domain_mapping = {"wdr.de": "https://www1.wdr.de"}
 
     def __init__(self, config):
         self.config = config
 
     def draw(self, content_dto: PaServiceContentItemDto, card):
-
         base_card_objects = [
             pn.pane.Markdown(f"""
                        #### {content_dto.title}
-                       **Datentyp:** {content_dto.contentType} 
+                       **Datentyp:** {self.type_icon.get(content_dto.contentType, content_dto.contentType)} 
+                       **Datum:** {content_dto.availableFrom}
                        **Strukturpfad:** {content_dto.structureNodePath}
-                       **External ID:** {content_dto.externalId} 
-                       **Sophora ID:** {content_dto.sophoraId}
+                       **External ID:** [{content_dto.externalId}]({self.config["reco_explorer_url_base"]}?externalId={content_dto.externalId}) 
```

Comment:
Hier gibt es ein Problem wenn mehrere Configs vorhanden sind (Mandanten Switcher) weil der das auch über URL Parameter steuert. Wenn eine nicht Start Config ausgewählt ist, wird die onclick hier überschrieben und man landet wieder beim Start Mandanten.

Reply:
Ja das stimmt.
Genau genommen werden einfach alle url parameter zurückgesetzt.
Da das aktuell eh erst mal ohnehin ein workaround ist, ist die Frage ob wir die Zeit investieren wollen, hier den State irgendwie in die Karte zu bekommen.